openbook_auth/models.py

```diff
@@ -62,6 +62,12 @@ class User(AbstractUser):
     )
 
     uuid = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)
+    JWT_TOKEN_TYPE_CHANGE_EMAIL = 'CE'
+    JWT_TOKEN_TYPE_PASSWORD_RESET = 'PR'
+    JWT_TOKEN_TYPES = (
```

Comment:
What does this do?

Reply:
Oh no, that was the idea but tuples dont work that way for constants.. so now i just use two constants. Forgot to remove this.